### PR TITLE
Simplified null return

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | A return statement wishing to return void should not return null.
| Type?         | improvement
| Category?     | CO
| BC breaks?    | Does it break backward compatibility? o
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | /
| How to test?  | /

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11861)

### CODE REVIEW

File: tests-legacy/Unit/classes/CacheCoreTest.php [hunk 1]
```diff
@@ -62,7 +62,7 @@ public function getFromArray()
         if (isset($this->cacheArray[$args[0]])) {
             return $this->cacheArray[$args[0]];
         } else {
-            return null;
+            return;
         }
```
Comment (eternoendless) [inline tests-legacy/Unit/classes/CacheCoreTest.php:66]: The whole `else` statement can go away

**Review by eternoendless — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
I would see the current `return null` as an explicit and wanted statement. Removing `null` or the whole `return null` could be considered as a forgotten case by some IDE linters.

**Comment by MathiasReker** [comment]
Should I close this PR?

**Comment by MathiasReker** [comment]
Do you want me to finish this PR or should I close it? :)

**Comment by eternoendless** [comment]
Thanks for your contribution @MathiasReker but I don't think it's worth spending more time on this PR.

**Pull request closed without merging**